apps/expo-go/ios/Exponent/Versioned/Core/EXVersionManagerObjC.mm

```diff
@@ -45,7 +47,7 @@
 #endif
 
 // Import 3rd party modules that need to be scoped.
-#import <RNCAsyncStorage/RNCAsyncStorage.h>
+#import "RNCAsyncStorage/RNCAsyncStorage.h"
```

Comment:
```suggestion
#import <RNCAsyncStorage/RNCAsyncStorage.h>
```

if you remove the scoped async-storage inside expo-go and use the one from node_modules, it would be more semantic correct to use `#import <...>` format